docs/book/features/volume_expansion.md

```diff
@@ -31,24 +40,105 @@ allowVolumeExpansion: true
 
 Proceed to create/edit a PVC by using this storage class.
 
-## Expand PVC Example
+## Expand PVC
+
+Prior to increasing the size of a PVC make sure that the PVC is in `Bound` state. If you are using a statically created PVC, ensure that the PVC and the PV specs have the `storageClassName` parameter pointing to a storage class which has `allowVolumeExpansion` set to true.
```

Comment:
nit: "statically created" -> "statically provisioned"

Reply:
Changed it, thanks!